test/helpers/vertical_nav_helper_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class VerticalNavHelperTest < ActionView::TestCase
+
+  test '#backend_api_nav_sections' do
+    @backend_api = FactoryBot.create(:backend_api)
+
+    # if permitted
+    stubs(can?: true)
```

Comment:
This stub is not bad but it feels a bit off to me. So just asking, this depends on what you are testing here. Does it matter to know what are the exact permissions or not? If not then leave it. If yes, you will need to include cancan controller additions and helpers and define current_user and the correct user.

Reply:
yes permission is not needed here